server/src/main/java/io/crate/planner/node/ddl/DeleteAllPartitions.java

```diff
@@ -23,31 +23,24 @@
 
 import java.util.List;
 
-import org.elasticsearch.action.admin.indices.delete.DeleteIndexAction;
-import org.elasticsearch.action.admin.indices.delete.DeleteIndexRequest;
-import org.elasticsearch.action.support.IndicesOptions;
-
-import io.crate.data.InMemoryBatchIterator;
 import io.crate.data.Row;
 import io.crate.data.Row1;
 import io.crate.data.RowConsumer;
-import io.crate.data.SentinelRow;
+import io.crate.execution.ddl.tables.DropPartitionsRequest;
+import io.crate.execution.ddl.tables.TransportDropPartitionsAction;
 import io.crate.execution.support.OneRowActionListener;
+import io.crate.metadata.RelationName;
 import io.crate.planner.DependencyCarrier;
 import io.crate.planner.Plan;
 import io.crate.planner.PlannerContext;
 import io.crate.planner.operators.SubQueryResults;
 
 public final class DeleteAllPartitions implements Plan {
```

Comment:
Looks like we could replace this plan's usages with a `DropTablePlan` and get rid of it?

Reply:
Ah probably not, as we only want to drop all partitions but not the table itself. so ignore ;)